### DISCUSSION TITLE
Can I run google ads on my github subdomain website?

### DISCUSSION THREAD

**Discussion by LhymeFruit** [community/community#70253] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Im running a website off github for my game servers. i would like to know if i was allowed to run google ads on my subdomain webpage. I would like to run ads off on the side or bottom of the page just for a passive income to help pay the electricity and internet bill. I have searched around for anyone else asking this and i only found old and outdated posts on 3rd party websites. I figured i'd ask here again because of the TOS changes.

**Comment by ktauchathuranga** [comment]
simple answer is "yes", according to few forums on web,

refer these links,
[Can I put Google Adsense Ads on a website hosted on GitHub Pages?](https://www.quora.com/Can-I-put-Google-Adsense-Ads-on-a-website-hosted-on-GitHub-Pages)
[Add advertising on a site loaded with github pages](https://github.com/orgs/community/discussions/22016)
[My website is hosted on GitHub pages](https://support.google.com/adsense/thread/129043738/my-website-is-hosted-on-github-pages?hl=en)

**Comment by Armoghan-ul-Mohmin** [comment, marked as answer]
Yes, you can run Google Ads on your subdomain webpage, as long as you own and control the subdomain, and your ads comply with all Google Ads policies.

**Comment by oussamaef06** [comment]
GitHub Pages, which allows you to host websites directly from GitHub repositories, had certain terms of service and usage policies in place. These policies did not explicitly address running third-party ads or monetization on GitHub Pages sites.

It's important to consider the most recent GitHub policies and terms of service, as they may have been updated or changed since my last update. Here are some general guidelines to keep in mind when considering running ads on your GitHub Pages site:

Check GitHub's Terms of Service: Review GitHub's current terms of service, as well as any specific policies or guidelines related to GitHub Pages. Ensure you comply with their terms to avoid any potential issues.

Consider the Purpose of Your Site: GitHub Pages is primarily intended for hosting personal websites, blogs, documentation, and project pages. The introduction of commercial activities, including running ads, could potentially go against the intended use.

Monetization and Ad Networks: If you plan to run ads, you should consider using reputable ad networks that comply with their terms and policies. Be cautious about the quality and relevance of ads you display, as low-quality or misleading ads may negatively affect the user experience.

Transparency and User Experience: Be transparent about the use of ads on your site, and ensure that ads do not significantly impact the user experience. Excessive or intrusive ads may lead to a poor user experience and could potentially violate GitHub's policies.

Check for Updates: GitHub's policies can change, so periodically check for any updates or changes in their terms of service or guidelines related to GitHub Pages.

Seek Legal Advice if Uncertain: If you're unsure whether running ads on your GitHub Pages site complies with GitHub's terms of service or if you have specific concerns, it's a good idea to consult with legal counsel for guidance.

To get the most accurate and up-to-date information, visit GitHub's official documentation, terms of service, or contact GitHub support directly for any specific inquiries related to monetization and running ads on GitHub Pages. Their support team can provide guidance based on the most current policies and guidelines in place.

**Reply by LhymeFruit** [reply]
why would i seek legal advice if i could just ask github.

**Comment by CGRoffcialcode** [comment]
im in a difficult spot too, trying to find someone who can support https://cgroffcialcode.github.io/The-Day-of-the-dead/DAy%20of%20dead%20htm%20code%201.html for ads. And basically no one wants to deal with the website. They see this spot  https://cgroffcialcode.github.io, and thats it

**Comment by Imadnajam** [comment]
t's recommended to use a hosting service that allows commercial activities and provides greater flexibility and control over your content.
